docs/configuration-reference.md

```diff
@@ -9,7 +9,8 @@ The Accelerated Bridge CNI configures networks through a CNI spec configuration
 * `ipam` (dictionary, optional): IPAM configuration to be used for this network.
 * `deviceID` (string, required): A valid pci address of a SWITCHDEV NIC's VF. e.g. "0000:03:02.3".
 * `debug` (bool, optional): Enable verbose logging
-* `bridge` (string, optional): Linux Bridge to use. e.g. `br1`, default value is `cni0`
+* `bridge` (string, optional): comma separated list of Linux Bridges to use. e.g. `br1, br2`, default value is `cni0`.
```

Comment:
a point of thought (and i know we discussed it before) but it had occurred to me that if we for some reason fail (e.g because of some uplink configuration we did not cover) to determine the uplink is connected to the bridge then this CNI is rendered inoperable.

as a safe guard we could keep `bridge` configuration as today and add a new `bridges` configuration.
these two (`bridge`, `bridges`) can be mutually exclusive. if both empty then `cni0` is chosen by default.

if `bridges` is provided then we do the bridge from uplink logic.

in cache we will still use conf.Bridge to save the actual bridge as of today.

as an example, with this PR, if user decides to tag/doubleTag/TrippleTag uplink for whatever reason or tag uplinks then bond them or bond uplinks and then tag bonds and connect it to the bridge CNI would just fail.

with what i propose (split the config). we can still accomodate those configuration when explicit bridge is specified.

Reply:
then for `bridges` option we document what are the supported configurations that will facilitate "auto-detection"

Reply:
discussed offline
decision is following:
1. keep single configuration option - bridge
2. bridge option should support bridge list in form "bridge1, bridge2"
3. if bridge option has no value (default will be used) or has a single value - skip auto detection logic (no check that uplink is a part of a bridge)
4. if multipile bridges provided - use auto detection logic (requires uplink to be a part of one of the bridges)